cymetric/execution.py

```diff
@@ -70,9 +74,11 @@ def __ne__(self, other):
         return self
 
 
-stripper = lambda s: s.strip()
+def stripper(s): return s.strip()
```

Comment:
Does PEP8 have a problem with `lambda` functions? Or is this a different kind of change?

Reply:
apparently yes:
```
PEP 207 indicates that reflexivity rules are assumed by Python. 
Thus, the interpreter may swap y > x with x < y, y >= x with x <= y, 
and may swap the arguments of x == y and x != y. The sort() and 
min() operations are guaranteed to use the < operator and the 
max() function uses the > operator. However, it is best to 
implement all six operations so that confusion doesn't arise in 
other contexts.

Always use a def statement instead of an assignment statement 
that binds a lambda expression directly to an identifier:

# Correct:
def f(x): return 2*x
# Wrong:
f = lambda x: 2*x
The first form means that the name of the resulting function object is 
specifically 'f' instead of the generic '<lambda>'. This is more useful 
for tracebacks and string representations in general. The use of the 
assignment statement eliminates the sole benefit a lambda expression 
can offer over an explicit def statement (i.e. that it can be embedded 
inside a larger expression)
```